internals/config/default.yml

```diff
@@ -0,0 +1,252 @@
+---
+name: "default name"
+slug: "default"
+
+settings:
+  mobile_phone_verification: false
+  subscriptions: false
+
+# configuration variables for the server app
+server:
+  # put the URL of openwisp-radius here
+  host: "http://127.0.0.1:8000"
+  proxy_urls:
+    password_change: "/api/v1/{org_slug}/account/password/change"
+    password_reset: "/api/v1/{org_slug}/account/password/reset"
+    password_reset_confirm: "/api/v1/{org_slug}/account/password/reset/confirm"
+    registration: "/api/v1/{org_slug}/account"
+    user_auth_token: "/api/v1/{org_slug}/account/token"
+    validate_auth_token: "/api/v1/{org_slug}/account/token/validate"
+    authorize: "/api/v1/authorize"
+    user_radius_sessions: "/api/v1/{org_slug}/account/session"
+    plans: '/api/v1/{org_slug}/plans'
+    create_mobile_phone_token: '/api/v1/{org_slug}/account/phone/token'
+    verify_mobile_phone_token: '/api/v1/{org_slug}/account/phone/verify'
+    mobile_phone_number_change: '/api/v1/{org_slug}/account/phone/change'
```

Comment:
one of the goals is to avoid this list of URLs.
According to a comment from some time ago, this should not be necessary: https://github.com/openwisp/openwisp-wifi-login-pages/issues/59

The ideal solution is to have these URLs defined only once for all organizations, without the need of redefining these URLs in each configuration of each organization, not only in the YAML, but I think we can avoid to having them defined in the config.json as well because that occupies space and needs to be downloaded, it doesn't make sense to repeat this over and over because it's always going to be the same.